frontend/src/metabase/admin/databases/containers/DatabaseEditApp.jsx

```diff
@@ -17,15 +18,14 @@ import * as databaseActions from "../database";
 
 const mapStateToProps = (state, props) => {
     return {
-        databaseId:       state.router && state.router.params && state.router.params.databaseId,
+        databaseId:       props.params.databaseId,
         database:         getEditingDatabase(state),
-        formState:        getFormState(state),
-        onChangeLocation: props.onChangeLocation
+        formState:        getFormState(state)
     }
 }
 
 const mapDispatchToProps = {
-    ...databaseActions
+    ...databaseActions,
```

Comment:
Missed a trailing comma here. 

Although I'm actually partial to the idea of keeping trailing comma's when args are vertically aligned like this, but either way we should apply it consistently.